packages/plans-grid-next/src/components/features-grid/top-buttons.tsx

```diff
@@ -8,8 +8,8 @@ import {
 import classNames from 'classnames';
 import { useTranslate } from 'i18n-calypso';
 import { GridPlan, PlanActionOverrides } from '../../types';
-import PlanFeatures2023GridActions from '../actions';
-import PlanDivOrTdContainer from '../plan-div-td-container';
+import PlanActions from '../shared/plan-actions';
```

Comment:
Similarly. Maybe just `actions`? Let me know what you think :-)

Reply:
Sounds good 👍